### ISSUE TITLE
No valid user token installed even after a full reset

### ISSUE THREAD

**Issue by kirabaen** [Razviar/mtga-pro-tracker#348]
I logged back into mtgarena.pro and was prompted to update my nick due to account wipe. I did so and then installed a new copy of the tracker from github (I had previously uninstalled it during the DDoS issues). I wasn't getting any info in the new tracker, so I did a full reset and copied my token from here (I've colored out most of the token in the image):

![mtgapro](https://user-images.githubusercontent.com/5413506/53616123-3837b000-3baf-11e9-985b-a27dcc0039af.PNG)

I then pasted the token into the tracker application but it's still giving me the message "No valid user token installed:"

![protracker](https://user-images.githubusercontent.com/5413506/53616118-3241cf00-3baf-11e9-9a11-556591fe04a7.PNG)

I've tried a few resets, running as administrator, etc.

Any ideas on what I should do to fix this and get the tracker working again?

Thanks!

**Comment by Aquaman88** [comment]
Thank you for your message. You can try to setup your profile again . You can input nick and all other parameters here: https://mtgarena.pro/edit-profile/ . You can try to add token and check your account name in tracker after that .

**Comment by kirabaen** [comment]
Thanks for the update.

I forgot to mention in my original post, but I did do that as well.

I have already re-entered email, username, and my account password. I didn't fill in the other fields because I don't really use facebook/twitter/etc.

My account token at https://mtgarena.pro/mtga-pro-tracker/ does not change after re-entering profile info (not sure if that is expected or not but wanted to mention it just in case).

The tracker shows the same token that I see on https://mtgarena.pro/mtga-pro-tracker/ in the MTGArena.pro token box, but still says "No valid user token installed!"

Is it possible to change/regenerate my account's token somehow?

Thank you.

**Comment by Razviar** [comment]
When you input the token, do you see any status messages right under input window? Like "no user found" or "check your token"? It seems that app just can't establish connection to server for some reason...

**Comment by kirabaen** [comment]
It just has a red circle and "No valid user token installed!"

After pasting the token in I've tried hitting enter and also tab - no change.

I've also typed the token in (and pressed enter and tab) but it still just says "No valid user token installed!"

**Comment by kirabaen** [comment]
Okay, I fixed this. There was something wrong with the installation for some reason.

As I mentioned initially I had uninstalled the application but I didn't really spot check it to make sure everything was removed cleanly at that time. I then just reinstalled and there must have been something cached from the original install that was causing an issue. Here is what I did to fix:

-Uninstalled MTGA Pro Tracker
-Deleted C:\Program Files (x86)\MTGA Pro Tracker (the uninstaller didnt remove this)
-Deleted MTGAProTracker folder from %AppData% local, locallow, and roaming app data folders (this was only present in two of them but I don't recall which - just check them all 
-Deleted the MTGAProTracker registry hive in HKLM\Software\ and HKCU\Software and HKUsers\<my user SID>\Software
-Deleted all previously downloaded installation files
-Downloaded  afresh copy from GitHub and extracted fully
-Extracted the MSI installer and ran it
-Installed to a non-default directory (I don't think this was necessary but I wanted to place it on a different drive anyway. In my case I installed to E:\Program Files (x86)\MTGA Pro Tracker\ instead.
-Made sure MTGA was not running and then launched the tracker application.
-Pasted my token from the website into the tracker and this time it found my account

Everything appears to be working normally after the above steps.

Thanks!
